app/views/artists/_artist.html.erb

```diff
@@ -1,5 +1,9 @@
   <div class="card shadow rounded me-3 mb-2 mt-2" style="width: 18rem;">
-  <img src=<%= artist.image_url %> class="even-size-artist" alt="...">
+     <% if artist.avatar.attached? %>
+       <%= image_tag artist.avatar, class: "even-size-artist" %>
+    <% else %>
+       <img src=<%= "https://dancewise.s3.amazonaws.com/misc-images/Blank+Avatar.png" %> class="even-size-artist">
```

Comment:
for blank avatars, we might want to just point to a local static image as part of the repo. but this works too.

Reply:
For a local image when we are in production, where would those images live?

Reply:
it would live in the code assets.